src/caselawclient/Client.py

```diff
@@ -625,6 +627,14 @@ def user_can_view_unpublished_judgments(self, username):
         result = multipart_data.parts[0].text
         return result.lower() == "true"
 
+    def user_can_view_unpublished_judgments_cached(self, username):
```

Comment:
Personally I wouldn't have a separate `_cached` implementation - probably I would have change `user_can_view_unpublished_judgements` to a private method and rename this one to that name. External users of the class don't need to care about the uncached version. But that's just personal OO style, it's certainly not a *necessary* change.